requirements/posix.txt

```diff
@@ -1,3 +1,3 @@
-ipykernel
 cloudpickle
+ipykernel
```

Comment:
It is easier to find the packages if they are sorted alphabetically

Reply:
Agreed.